python/dgllife/utils/featurizers.py

```diff
@@ -1369,6 +1371,78 @@ def __init__(self, atom_data_field='h'):
                  atom_chirality_type_one_hot]
             )})
 
+class PAGTN_AtomFeaturizer(BaseAtomFeaturizer):
+    """The Atom featurizer used in PAGTN
+
+    PAGTN is introduced in
+    `Path-Augmented Graph Transformer Network. <https://arxiv.org/abs/1905.12712>`
+
+    The atom features include:
+
+    * **One hot encoding of the atom type**.
+    * **One hot encoding of Formal charge of the atom**.
+    * **One hot encoding of the atom degree**
+    * **One hot encoding of explicit valence of an atom**. The supported possibilities
+      include ``0 - 6``.
+    * **One hot encoding of implicit valence of an atom**. The supported possibilities
+      include ``0 - 5``.
+    * **Whether the atom is aromatic**.
+
+    Parameters
+    ----------
+    atom_data_field : str
+        Name for storing atom features in DGLGraphs, default to 'h'.
+
+    Examples
+    --------
+
+    >>> from rdkit import Chem
+    >>> from dgllife.utils import PAGTN_AtomFeaturizer
+
+    >>> mol = Chem.MolFromSmiles('C')
+    >>> atom_featurizer = PAGTN_AtomFeaturizer(atom_data_field='feat')
+    >>> atom_featurizer(mol)
+    {'feat': tensor([[1., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+                      0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+                      0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+                      0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 1., 0., 0.,
+                      0., 0., 0., 0., 0., 0., 0., 0., 1., 0., 0., 0., 0., 0., 0., 0., 0., 0.,
+                      0., 1., 0., 0.]])}
+    >>> # Get feature size for nodes
+    >>> print(atom_featurizer.feat_size())
+    94
+
+    See Also
+    --------
+    BaseAtomFeaturizer
+    CanonicalAtomFeaturizer
```

Comment:
Missing `WeaveAtomFeaturizer`. Also can you add `PAGTN_AtomFeaturizer` in `See Also` of other atom featurizers?